File: adventofcode-2024/src/test/java/nl/mvdr/adventofcode/adventofcode2024/day14/Part1Test.java
package nl.mvdr.adventofcode.adventofcode2024.day14;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class Part1Test {

    @Test
    void testExample() {
        Part1 solver = new Part1(11, 7);
        
        var result = solver.solve("example-day14-2024.txt");
        
        Assertions.assertEquals("12", result);
    }
}

File: adventofcode-2025/src/test/kotlin/nl/mvdr/adventofcode/adventofcode2025/day08/Part1Test.kt
package nl.mvdr.adventofcode.adventofcode2025.day08

import nl.mvdr.adventofcode.solver.FunctionSolver
import org.junit.jupiter.api.Test
import kotlin.test.assertEquals

class Part1Test {

    @Test
    fun testExample() {
        val solver = FunctionSolver { solvePart1(it, 10) }

        val result = solver.solve("example-day08.txt")

        assertEquals("40", result)
    }
}
